pages/index.js

```diff
@@ -17,12 +18,12 @@ export default function Home({
   onShowPassword,
   showConfirmedPassword,
   onShowConfirmedPassword,
-  onSignIn,
-  users,
 }) {
+  const {user, setUser} = useContext(UserContext);
+  console.log(user);
   return (
     <>
-      {!signedIn ? (
+      {user === null || user.length === 0 ? (
         <>
```

Comment:
`user` is now an object and objects don't have a `length` property. So please remove this check.